tests/test_audio.py

```diff
@@ -28,14 +28,15 @@
 
 import telegram
 from tests.base import BaseTest, timeout
+from tests.reupload import get_file_id
 
 
 class AudioTest(BaseTest, unittest.TestCase):
     """This object represents Tests for Telegram Audio."""
 
     def setUp(self):
         self.audio_file = open('tests/data/telegram.mp3', 'rb')
-        self.audio_file_id = 'CQADAQADDwADHyP1B6PSPq2HjX8kAg'
+        self.audio_file_id = get_file_id('audio')
```

Comment:
This would upload the file for every test in test_audio. setUp() is called before every test. I would go for [setUpClass](https://docs.python.org/2/library/unittest.html#unittest.TestCase.setUpClass) in this case